native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -173,11 +173,13 @@ enum class Feature : uint32_t {
     MULTIPLE_RENDER_TARGETS,
     BLEND_MINMAX,
     COMPUTE_SHADER,
-    // @deprecated
-    INPUT_ATTACHMENT_BENEFIT,
+
+    INPUT_ATTACHMENT_BENEFIT, // @deprecated
```

Comment:
Deprecate like this will not trigger warning message.

Reply:
I tried another way. Will it trigger now?

Reply:
You can use `CC_DEPRECATED()` instead. It works on functions and classes. I think it can work here too.

Reply:
It seems CC_DEPRECATED does not work on enum value.